src/Tizen.Network.Stc/Tizen.Network.Stc/StcInfo.cs

```diff
@@ -0,0 +1,238 @@
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get AppId from Info, Error - " + (StcError)ret);
+                    return "";
+                }
+                return appId;
+            }
+        }
+
+        /// <summary>
+        /// A property to get interface name from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
+        /// <value>Interface name.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public string IfaceName
+        {
+            get
+            {
+                string ifaceName;
+                int ret = Interop.Stc.Info.GetIfaceName(_infoHandle, out ifaceName);
+                if (ret != (int)StcError.None)
+                {
+                    Log.Error(Globals.LogTag, "Failed to get interface name from info, Error - " + (StcError)ret);
+                    return "";
+                }
+                return ifaceName;
+            }
+        }
+
+        /// <summary>
+        /// Get the time interval from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
+        /// <param name="from">The beginning of the time interval.</param>
+        /// <param name="to">The end of the time interval.</param>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown when the Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when the method failed due to an invalid operation.</exception>
+        public void GetTimeInterval(out long from, out long to)
+        {
+            int ret = Interop.Stc.Info.GetTimeInterval(_infoHandle, out from, out to);
+            if (ret != (int)StcError.None)
+            {
+                Log.Error(Globals.LogTag, "Failed to get time interval from info, Error - " + (StcError)ret);
+                StcErrorFactory.ThrowStcException(ret);
+            }
+        }
+
+        /// <summary>
+        /// A property to get the interface type from statistics information.
+        /// </summary>
+        /// <since_tizen> tizen_5.5 </since_tizen>
+        /// <value>Interface type.</value>
+        /// <privilege>http://tizen.org/privilege/network.get</privilege>
+        /// <exception cref="NotSupportedException">Thrown while setting this property when Stc is not supported.</exception>
+        /// <exception cref="InvalidOperationException">Thrown while setting this value due to an invalid operation.</exception>
+        public StcIfaceType Iface
```

Comment:
X DO NOT use abbreviations or contractions as part of identifier names.
https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/general-naming-conventions

Reply:
Done.